internal/injector/config/config-yml.go

```diff
@@ -0,0 +1,150 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package config
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/DataDog/orchestrion/internal/injector/aspect"
+	"gopkg.in/yaml.v3"
+)
+
+const OrchestrionYML = "orchestrion.yml"
+
+// loadYMLFile loads configuration from the specified directory.
+func (l *Loader) loadYMLFile(dir string, name string) (*configYML, error) {
+	filename := name
+	if !filepath.IsAbs(name) {
+		filename = filepath.Join(dir, name)
+	}
```

Comment:
Why do this on yml files and not go files ? Is there a test case covering this ?

Reply:
`loadGoFile` is _always_ called with an absolute filename... Whereas `loadYMLFile` is (at least currently) always called with a relative filename (and it's called from the Go loader, so it double-checks for safety).

Those lines are covered by tests.